### PR TITLE
Add migration for notifications create_on row

### PR DESCRIPTION
**Pull request by Petitoto**
### Description

Add migration for #371

### Checklist

- [ ] Created tests which fail without the change (if possible)
- [x] All tests passing
- [ ] Extended the documentation, if necessary

### CODE REVIEW

File: migrations/versions/7-notifications_create_on.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+"""
+
+from collections.abc import Sequence
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "e3d06397960d"
```
Comment (julien4215) [inline migrations/versions/7-notifications_create_on.py:15]: The revision should be the one of phonebook
```suggestion
revision: str = "6afc765adaa2"
```

**Review by julien4215 — CHANGES_REQUESTED** [review]
Nice migration file !
Do we need the `existing_type` and `existing_nullable` parameter in `alter_column` ? It seems only required for MySQL according to the documentation.

File: migrations/versions/7-notifications_create_on.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+
+def upgrade() -> None:
+    op.alter_column(
+        "notification_message",
+        "expire_on",
+        existing_type=sa.DATE(),
+        type_=sa.DateTime(timezone=False),
+        existing_nullable=False,
+    )
```
Comment (julien4215) [inline migrations/versions/7-notifications_create_on.py:28]: Nice migration file !
Do we need the existing_type and existing_nullable parameter in alter_column ? It seems only required for MySQL according to the documentation.

**Review by julien4215 — APPROVED** [review]

File: migrations/versions/7-notifications_create_on.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+
+def downgrade() -> None:
+    op.alter_column(
+        "notification_message",
+        "expire_on",
+        existing_type=sa.DateTime(timezone=False),
+        type_=sa.DATE(),
```
Comment (julien4215) [inline migrations/versions/7-notifications_create_on.py:36]: Maybe we could use the CamelCase datatype ?

**Review by julien4215 — CHANGES_REQUESTED** [review]
Nice migration file !
Do we need the existing_type and existing_nullable parameter in alter_column ? It seems only required for MySQL according to the documentation.

**Review by guillaumetavernier — APPROVED** [review]

File: migrations/versions/7-notifications_create_on.py (hunk 3 above)
Comment (Petitoto) [reply migrations/versions/7-notifications_create_on.py]: I'm not sure to understand the difference. I commited the change, but did not test.

**Review by Foukki — APPROVED** [review]

File: migrations/versions/7-notifications_create_on.py (hunk 3 above)
Comment (julien4215) [reply migrations/versions/7-notifications_create_on.py]: The uppercase type DATE is the SQL type DATE whereas Date in CamelCase is an agnostic database type meaning that SQLAlechemy will use the the most appropriate SQL type depending on the database. In the documentation they give the exemple of the Boolean that will render BOOLEAN for PostgreSQL, BIT for MySQLand SMALLINT for Oracle.

**Review by julien4215 — APPROVED** [review]

**Review by Rotheem — APPROVED** [review]

**Pull request merged**